warehouse/migrations/versions/1dbb95161e5a_add_vulnerability_and_release_.py

```diff
@@ -0,0 +1,70 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Add Vulnerability and release association tables
+
+Revision ID: 1dbb95161e5a
+Revises: 08ccc59d9857
+Create Date: 2021-07-14 19:01:16.356312
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "1dbb95161e5a"
+down_revision = "08ccc59d9857"
+
+# Note: It is VERY important to ensure that a migration does not lock for a
```

Comment:
comment(s) from template can be removed (not critical)

Reply:
done